package.json

```diff
@@ -16,6 +16,16 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
+		"viewsWelcome": [
+      {
+        "view": "OneExplorerView",
+        "contents": "Open folder where model files are stored. Models files are one of the following format:\n- *.tflite, *.pb, Keras model or saved model for Tensorflow​\n- *.onnx or *.pth for PyTorch\n[Open Folder](Command:onevscode.fileTreeView-open)\n"
```

Comment:
```suggestion
        "contents": "Open folder where model files are stored.\nModel files are one of the following format:\n- *.tflite, *.pb, Keras model or saved model for Tensorflow​\n- *.onnx or *.pth for PyTorch\n[Open Folder](Command:onevscode.fileTreeView-open)\n"
```

Reply:
Does `Keras model` or `saved model` and `.pth` are visible in one-explorer? I read the codes, but I'm not sure about it.
/cc @dayo09